frappe/public/scss/desk/report.scss

```diff
@@ -84,14 +84,37 @@
 	margin-bottom: 10px;
 }
 
+.layout-main-section .frappe-card {
+	--report-filter-height: 0px;
+}
+
 .report-wrapper {
 	overflow: auto;
+
+	.datatable {
+		height: calc(100vh - var(--report-filter-height) - 205px);
```

Comment:
`205 px` and other px values seem arbitrary... how are we getting these values? can we instead have relative values to avoid unexpected UI on differently sized screens?

Reply:
I tested on different screen sizes and came up with that value.